CheckDrive.Web/CheckDrive.Web/Views/DoctorReviews/Index.cshtml

```diff
@@ -22,7 +20,7 @@
                 </div>
             </div>
             <!-- Date -->
-            <div class="col-md-2 mb-4">
+            <div class="col-md-2 mb-4 create-btn">
```

Comment:
I think the class name (**create-btn**) is a bit strange.
It would be better if the input type was given a suitable name.